tests/external_tests/gamepace/test_gamepace.py

```diff
@@ -3,65 +3,65 @@
 from mlbstatsapi import Mlb
 
 
-class TestGamepace(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls) -> None:
-        cls.mlb = Mlb()
+# class TestGamepace(unittest.TestCase):
```

Comment:
Is gamepace still causing issues?

Reply:
It seems to fail at random times that end point sucks.

Reply:
Cool, well keep i as a mock for now. That works for me until the mlb stats api can get their crap together and return a consistent result for the same query